First Backup Sync Implementation (#115)
nemtech/symbol-bootstrap: docs/backup.md

`symbol-bootstrap backup`
=========================

> The command backups the Mongo and RocksDb data folder into a Zip file that can be used for --backupSync feature. Bootstrap compose services must be stopped before calling this command.

Reviewer: ```suggestion
The command backs up the Mongo and RocksDb data folder into a Zip file that can then be used by the `--backupSync` feature. Bootstrap compose services must be stopped before calling this command.
```

> Note: this command is designed for NGL to be used when running public main or public test networks. It's not backing up any node specific information.

Reviewer: Is this true? Couldn't it be used by a private network to speed up sync?

Author: Yes, it can be used. Atm we don't have a well-documented process of a user joining a private network with bootstrap. Pieces are there, I just need to put them together. Main issue is how to easily share a private network preset and seed in new private nodes

Author: It's just a warning for the current users that may want to use it on testnet backups. This backup is not backing a node (like's node configuration or keys), it's just the data that can be shared between multiples nodes

* [`symbol-bootstrap backup`](#symbol-bootstrap-backup)

## `symbol-bootstrap backup`

> The command backups the Mongo and RocksDb data folder into a Zip file that can be used for --backupSync feature. Bootstrap compose services must be stopped before calling this command.

Reviewer: ```suggestion
The command backs up the Mongo and RocksDb data folder into a Zip file that can then be used by the `--backupSync` feature. Bootstrap compose services must be stopped before calling this command.
```

```
USAGE
  $ symbol-bootstrap backup

OPTIONS
  -h, --help                         It shows the help of this command.

>   -t, --target=target                [default: target] The target folder where the symbol-bootstrap network is generated

Reviewer: ```suggestion
  -t, --target=target                [default: target] The target folder where the symbol-bootstrap network is generated.
```
